client/src/components/PickupDetail.js

```diff
@@ -50,9 +50,9 @@ const PickupDetail: () => React$Node = () => {
         </div>
       </div>
       <div class="container-fluid m-1">
-        <a role="button" onClick={backToLanding} className="btn btn-primary mx-1">
+        <button type="button" onClick={backToLanding} className="btn btn-primary mx-1">
```

Comment:
What's the benefit of using button instead of a link?

Reply:
From my understanding, nothing big diffrence so far, I just want them to be consistent in case we want to add some styles through button tags in the future. Is that ok for you?